be/src/vec/exec/scan/scanner_context.cpp

```diff
@@ -174,11 +176,30 @@ bool ScannerContext::set_status_on_error(const Status& status) {
 
 Status ScannerContext::_close_and_clear_scanners() {
     std::unique_lock<std::mutex> l(_scanners_lock);
+    std::stringstream scanner_statistics;
```

Comment:
This is a hot path, better use `fmt`.
And print it only when `enable_profile` is true